### ISSUE TITLE
"No tweaks available" Nex 5

### ISSUE THREAD

**Issue by hnyavsik** [ma1co/Sony-PMCA-RE#261]
Cant tweak my NEX-5 on update mode

Initializing firmware update
Writing firmware
0%
100%
Cannot sync backup
Done

**Comment by n-kremeris** [comment]
I have the same issue - trying to start tweaking on a nex-5  shows "No tweaks available".

**Comment by ma1co** [comment]
This is correct, currently, there are no tweaks available for these old cameras :(

**Comment by Growl007** [comment]
I’ve also just discovered this issue with my Sony Nex-5 (Japanese model).

I was hoping for just a tweak to set the language to English!

**Comment by Growl007** [comment]
Wondering if the Nex-5n would have any tweaks or would it be the same as the original Nex-5

Wish there was a camera listing showing cameras and supported tweaks.
